packages/blockchain-wallet-v4-frontend/src/components/Flyout/index.tsx

```diff
@@ -156,28 +157,34 @@ export const StickyHeaderFlyoutWrapper = styled(FlyoutWrapper)`
   z-index: 99;
 `
 
-const Flyout = ({ children, isOpen, ...props }: Props) => {
-  return (
-    <AnimatePresence>
-      {isOpen && !props.userClickedOutside ? (
-        <FlyoutModal
-          transition={{
-            bounce: 0,
-            type: 'spring'
-          }}
-          initial={{ x: width }}
-          animate={{ x: 0 }}
-          exit={{ x: width }}
-          {...props}
-        >
-          <FlyoutChildren>
-            {/* Each child must be wrapped in FlyoutChild for transitioning to work */}
-            {children}
-          </FlyoutChildren>
-        </FlyoutModal>
-      ) : null}
-    </AnimatePresence>
-  )
+class Flyout extends React.Component<Props> {
```

Comment:
This should be a functional component, wrapped in `memo` (Like `React.memo`), with this function: `(nextProps) => !equals(this.props, nextProps)` as a second parameter, like:

```tsx
export default memo(Flyout, (nextProps) => !equals(this.props, nextProps))
```

Then you don't need to use a class component